### DISCUSSION TITLE
Get started discussion

### DISCUSSION THREAD

**Discussion by oktaaa17** [openyourmind-indo/OpenYourMind-1#7] [General]
Section discussion, if have any questions

**Comment by alwan-syah** [comment]
![image](https://github.com/openyourmind-indo/OpenYourMind-1/assets/118163144/b0f9a563-a73e-4388-80e8-a980f6a21f0a)
[](url)
dimas, gua udah push bagian task gua tapi karena gua make fork baru ke push di repo gua, nah ini biar push nya bisa repo utama itu gimana? gua push nya di branch gua

**Reply by oktaaa17** [reply]
@alwan-syah kalau udah remote tinggal gini aja sih, git push -u origin `nama branch`

**Comment by oktaaa17** [comment]
ntar tinggal pr gabungin ke dev branch
